### PR TITLE
SQL/Database: Update Trial of the Champion Black Knight Gryphon waypoint.

### PR DESCRIPTION
**Pull request by sucofog**
**Changes proposed:** Update Trial of the Champion Black Knight Gryphon waypoint.

More Blizzlike Waypoint for npc_black_knight_skeletal_gryphon, NPC= 35491
Gryphon updated waypoint corrected the orientation when landing.
Data is DELETED and RE-INSERTED, becouse new waypoint has new 5 blizzlike waypoint.

**Target branch(es):** master

**Tests performed:**  Is tested and working

### CODE REVIEW

File: data/sql/updates/db_world/2016_12_07_00.sql [hunk 1]
```diff
@@ -0,0 +1,30 @@
+-- More Blizzlike Waypoint for npc_black_knight_skeletal_gryphon, NPC= 35491
+-- Gryphon updated waypoint corrected the orientation when landing.
+-- Data is DELETED and RE-INSERTED, becouse new waypoint has new 5 blizzlike waypoint.
```
Comment (ShinDarth) [inline data/sql/updates/db_world/2016_12_07_00.sql]: a note for future:  just use the PR description for this kind of comments, there is not need to put in the actual SQL file

Comment (sucofog) [inline data/sql/updates/db_world/2016_12_07_00.sql]: Allright!

**Comment by Krath** [comment]
I'm working on better waypoints, anyone want to help me?

**Comment by sucofog** [comment]
You tested @Krath ?

For me is correct, data is right

**Comment by Krath** [comment]
Yeah, but I'm working on more blizzlike waypoints

**Comment by sucofog** [comment]
Okay, talk me on gitter @Krath so we work on it

**Comment by sucofog** [comment]
I close due too rewriting, till i end this with Retail correct data.

**Comment by Yehonal** [comment]
@sucofog did you work on a valid PR for it?

**Pull request closed without merging**